### DISCUSSION TITLE
これまで上がってきた要望のトレンドを提供する

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#12] [要望・アイデア・不具合]
> 既に挙がっている意見に対して共感を示せる
> 運営からのメッセージに対してリアクションを示せる

これは会社に上がってきたユーザーの要望を公開することになるので、プライバシー権との兼ね合いにもなりますが、（なのであらかじめ要望投稿フォームに転載可能性について記載しておく）

Qiita って人気記事ランキングってのがあったりすると思います。
同様に、Qiita にこれまで挙げられた要望のトレンドとか出してみるのはどうでしょう？
かなり思い切った案ですので、賛否両論になりそう😱

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！

> Qiita にこれまで挙げられた要望のトレンド

現状、この役割をDiscussions機能で担保できればと考えています。
Qiita上はあくまで記事について、GitHub Discussions上では要望について、と住み分けつつも、気軽に両者を行き来できるような作りにしていくつもりでいます！
